tf_agents/cogment_verse_tf_agents/reinforce/reinforce.py

```diff
@@ -133,12 +133,17 @@ def reset_replay_buffer(self):
         self._replay_buffer._write_index = -1
         self._replay_buffer._n = 0
 
-    def init_params(self, model_parms):
-        self._model.set_weights(model_parms)
-        self._model.trainable = True
-
     def get_replay_buffer_size(self):
         """
         Return the size of the internal replay buffer
         """
         return self._replay_buffer.size()
+
+    def save(self, f):
+        self._params["model_params"] = self._model.get_weights()
+        torch.save(self._params, f)
```

Comment:
You are now just saving/loading a simple data structure you can switch to pickle (https://docs.python.org/3/library/pickle.html#examples) and get rid of torch here.